### DISCUSSION TITLE
[Question] Is it possible to configure bull to use a single key as a list for jobs in redis?

### DISCUSSION THREAD

**Discussion by luiscabus** [OptimalBits/bull#2753] [Q&A] [answered]
In case I am using redis for many things, the jobs keys makes things very polluted, requiring additional software/logic to read redis. 
In bee-queue, for example, only a few keys are used, with one containing all the jobs to be picked up. Laravel queue also does it that way.
But bull will use one key for each job, is it possible to configure it to "group" the jobs in one key/hash?

**Comment by manast** [comment, marked as answer]
I don't think BeeQueue is much different other than having fewer features as it was an in-official fork of Bull. It is not possible to implement the feature set and performance of Bull with much less key pollution than we do.

In any case, if you use Redis for other things and want isolation, there is a mechanism for that built-in in Redis which is that you can use different DBs, however, it is not supported by Redis Cluster: https://redis.io/docs/latest/commands/select/ You can easily choose the DB as a IORedis option when instantiating your queues: https://github.com/redis/ioredis?tab=readme-ov-file#connect-to-redis

Btw, if you are new to Bull I heavily recommend you to use BullMQ instead: https://github.com/taskforcesh/bullmq

**Reply by luiscabus** [reply]
Ok, thank you!
